build.sh

```diff
@@ -52,6 +52,9 @@ dockerbuild()
     BUILD_COMMAND=/opt/code/build_projects/dotnet-cli-build/build.sh $DIR/scripts/dockerrun.sh --non-interactive "$@"
 }
 
+# Initialize the tooling
+$DIR/init-tools.sh "${args[@]}"
```

Comment:
No that won't work, the init tools is still run in the outer machine context.

`BUILD_COMMAND=/opt/code/build_projects/dotnet-cli-build/build.sh` Is the script run in the docker context.